store.html

```diff
@@ -0,0 +1,157 @@
+                    <li class='nav-li'><a class='nav-link' href='#'>Help</a></li>
+                    <li class='nav-li'><a class='nav-link' href='#'>Login</a></li>
+                </ul> <!-- /na-ul -->
+            </div> <!-- /nav-flex-container -->
+        </nav>
+        
+
+        <!-- Main -->
+
+        <main class="store-main">
+
+            <section class='left-flex-item'>
+
+                <!-- Call to Action -->
+
+                <h1 class='left-flex-item-item store-cta'>Order your Karma wifi device today!</h1>
+
+
+                <!-- Form -->
+
+                <form class='left-flex-item-item' id='order-form'>
+
+                    
+                    <!-- for Accessibility -->
+
+                    <fieldset class="hide-this">
+                        <legend class='hide-this'>Your Name</legend>
+
+                        <div class='form-flex show-this'>
+
+                            <div class='width-475'>
+                            <label class='label' for="first-name">First name *</label>
+                            <input class='input' id='first-name' type='text' required minlength=2>
+                            </div>
+
+                            <div class='width-475'>
+                            <label class='label' for="last-name">Last name *</label>
+                            <input class='input' id='last-name' type='text' required minlength=2>
+                            </div>
+
+                        </div> 
+                    </fieldset>
+
+                    <fieldset class='hide-this'>
+                        <legend class='hide-this'>Your Address</legend>
+
+                        <div class='your-address show-this'>
+
+                            <label class='label' for='address-one'>Address *</label>
+                            <input class='input' type="text" id="address-one" required minlength=4>
+
+                            <label class='label' for='address-two'>Address 2</label>
+                            <input class='input' type="text" id="address-two">
+
+                            <div class='form-flex'>
+
+                                <div class='width-575'>
+                                    <label class='label' for='city'>City *</label>
+                                    <select class='select input' required id='city' form='order-form'>
+                                        <option value="" disabled selected>Select your city...</option>
+                                        <option value='london'>London</option>
+                                        <option value='budapest'>Budapest</option>
+                                        <option value='new-york'>New York</option>
+                                    </select>
+                                </div>
+                                
+                                <div class='width-375'>
+                                    <label class='label' for='postcode'>Postcode *</label>
```

Comment:
I have noticed this across the document, you should use double quotes instead of single quotes for attributes values
```suggestion
                                    <label class="label" for="postcode">Postcode *</label>
```